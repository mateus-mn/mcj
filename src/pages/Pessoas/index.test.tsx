import { render } from '@testing-library/react';
import { QueryClient, QueryClientProvider } from 'react-query';
import PessoasPage from '.';

describe('Testes da pÃ¡gina Pessoas', () => {
	it('Teste de snapshot', () => {
		const queryClient = new QueryClient();
		const { container } = render(
			<QueryClientProvider client={queryClient}>
				<PessoasPage />
			</QueryClientProvider>,
		);

		expect(container).toMatchSnapshot();
	});
});
