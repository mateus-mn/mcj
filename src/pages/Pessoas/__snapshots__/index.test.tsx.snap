// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes da p√°gina Pessoas Teste de snapshot 1`] = `
<div>
  <div
    class="container-fluid"
  >
    <h1
      class="text-center mt-3"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-user "
        data-icon="user"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0S96 57.3 96 128s57.3 128 128 128zm-45.7 48C79.8 304 0 383.8 0 482.3C0 498.7 13.3 512 29.7 512H418.3c16.4 0 29.7-13.3 29.7-29.7C448 383.8 368.2 304 269.7 304H178.3z"
          fill="currentColor"
        />
      </svg>
       Pessoas
    </h1>
    <button
      class="btn btn-primary btn-sm"
      disabled=""
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-plus "
        data-icon="plus"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
          fill="currentColor"
        />
      </svg>
       Nova pessoa
    </button>
    <div
      class="text-center"
    >
      <div
        class="spinner-border"
        role="status"
      >
        <span
          class="visually-hidden"
        >
           Carregando... 
        </span>
      </div>
    </div>
  </div>
</div>
`;
